src/lifecycles/CryptoSetupExtensions.ts

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2023 Verji Tech AS
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * Types copied (and renamed) from matrix-js-sdk
+ */
+
+export interface SecretStorageKeyDescriptionCommon {
+    /** A human-readable name for this key. */
+    // XXX: according to the spec, this is optional
```

Comment:
Link?

Reply:
There is a comment in the code which says the types were copied from the matrix-js-sdk. The comments were copied verbatim. 

So I could: 
A. Keep the comments as they are, but add a link to the source matrix-js-sdk file
B. Try to look up the spec, and add a link supporting the comment
C. B + also making a PR for matrix-js-sdk, updating the comment there 

It is not clear to me which one is the "best"?

Reply:
Opted for A in the new commits